macos_resources/jenkins-agent.plist

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Label</key>
+	<string>org.ros2.ci.REPLACE_HOSTNAME-agent</string>
+	<key>ProgramArguments</key>
+	<array>
+		<string>/usr/bin/java</string>
+		<string>-Djava.awt.headless=true</string>
+		<string>-jar</string>
+		<string>/Users/osrf/jenkins-agent/slave.jar</string>
+		<string>-jnlpUrl</string>
+		<string>http://ci.ros2.org/computer/REPLACE_JENKINS_AGENT_NAME/slave-agent.jnlp</string>
+		<string>-secret</string>
+		<string>REPLACE_JENKINS_AGENT_SECRET</string>
+	</array>
+	<key>KeepAlive</key>
+	<true/>
+	<key>ProcessType</key>
```

Comment:
is it correct to say that "The process is set to `Interactive` type and given high scheduling priority so that when ROS 2 tests are run those related to performance (timing etc) do not suffer."? Both are necessary for that right? If so I'll add that as a comment for whoever finds this without the context of this PR.

Reply:
> "The process is set to Interactive type and given high scheduling priority so that when ROS 2 tests are run those related to performance (timing etc) do not suffer

The process type is set to interactive so that the supervised process is afforded the same priority as the previous manually started agents.
The niceness value is really just for emphasis that this is process should be allowed to run with minimal hinderance on our build machines. It may not be necessary (manually spawned processes have the default niceness).

If we wanted performance tests to be representative of default background task behavior on MacOS we could drop these configuration keys and instead treat the resulting performance hit and possible perf-test failures as bugs.


> Both are necessary for that right?